core/src/test/scala/integration/kafka/api/BaseConsumerTest.scala

```diff
@@ -117,38 +116,29 @@ object BaseConsumerTest {
   // * KRaft and the classic group protocol
   // * KRaft and the consumer group protocol
   def getTestQuorumAndGroupProtocolParametersAll() : java.util.stream.Stream[Arguments] = {
-    util.Arrays.stream(Array(
-        Arguments.of("zk", "classic"),
-        Arguments.of("kraft", "classic"),
-        Arguments.of("kraft", "consumer")
-    ))
+    stream.Stream.of(
+      Arguments.of("zk", "classic"),
+      Arguments.of("kraft", "classic"),
+      Arguments.of("kraft", "consumer")
+    )
   }
 
-  // In Scala 2.12, it is necessary to disambiguate the java.util.stream.Stream.of() method call
-  // in the case where there's only a single Arguments in the list. The following commented-out
-  // method works in Scala 2.13, but not 2.12. For this reason, tests which run against just a
-  // single combination are written using @CsvSource rather than the more elegant @MethodSource. 
-  // def getTestQuorumAndGroupProtocolParametersZkOnly() : java.util.stream.Stream[Arguments] = {
```

Comment:
This should be used by `testAssignAndConsumeWithLeaderChangeValidatingPositions`. Since we are phasing out ZooKeeper, it's fine to remove it for now.

https://github.com/apache/kafka/blob/6ee46ea86530e019798ba0d4183164d7925dde63/core/src/test/scala/integration/kafka/api/PlaintextConsumerAssignTest.scala#L143

Reply:
I have uncommented the `getTestQuorumAndGroupProtocolParametersZkOnly` method and replaced `@CsvSource` with `@MethodSource`.